MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -1114,6 +1116,13 @@ void EnggDiffractionPresenter::doFocusing(const EnggDiffCalibSettings &cs,
   }
   g_log.notice() << "Saved focused workspace as file: " << fullFilename
                  << std::endl;
+
+  bool saveOutputFiles = m_view->saveOutputFiles();
+  if (saveOutputFiles) {
```

Comment:
@FedeMPouzols: Would you want the previous [nexus file saved](https://github.com/mantidproject/mantid/blob/13854_Engg_save_focused_data_in_3formats/MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp#L1102-L1118) in this `if` statement as well?